src/sentry/south_migrations/0289_populate_release_project_new_groups.py

```diff
@@ -0,0 +1,726 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        db.commit_transaction()
+
+        for release in RangeQuerySetWrapperWithProgressBar(orm.Release.objects.all()):
```

Comment:
Let's qualify this with `.exclude(new_groups=0)` since these rows require no work to be done. And oddly, this excludes 2/3rds of our production data.